src/Compiler/Lexer.cpp

```diff
@@ -45,6 +45,7 @@ namespace Ark::internal
             {
                 line++;
                 character = 0;
+                continue;
```

Comment:
I'm having a problem with this, because if we are in a comment:
```clojure
# test
# bla
(let c ((12)))
(let a (12))
```
the comment is never closed, thus everything afterward get appended in the comment

Reply:
Ok, resolved!